[2025-06-01T00:00:04.293+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: real_estate_pipeline.generate_data scheduled__2025-05-31T00:00:00+00:00 [queued]>
[2025-06-01T00:00:04.326+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: real_estate_pipeline.generate_data scheduled__2025-05-31T00:00:00+00:00 [queued]>
[2025-06-01T00:00:04.327+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-06-01T00:00:04.379+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): generate_data> on 2025-05-31 00:00:00+00:00
[2025-06-01T00:00:04.411+0000] {standard_task_runner.py:60} INFO - Started process 3804 to run task
[2025-06-01T00:00:04.425+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'real_estate_pipeline', 'generate_data', 'scheduled__2025-05-31T00:00:00+00:00', '--job-id', '32', '--raw', '--subdir', 'DAGS_FOLDER/real_estate_pipeline.py', '--cfg-path', '/tmp/tmpj0wh0_r_']
[2025-06-01T00:00:04.436+0000] {standard_task_runner.py:88} INFO - Job 32: Subtask generate_data
[2025-06-01T00:00:04.531+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-06-01T00:00:04.654+0000] {task_command.py:423} INFO - Running <TaskInstance: real_estate_pipeline.generate_data scheduled__2025-05-31T00:00:00+00:00 [running]> on host e80848f7052f
[2025-06-01T00:00:04.939+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='real_estate_pipeline' AIRFLOW_CTX_TASK_ID='generate_data' AIRFLOW_CTX_EXECUTION_DATE='2025-05-31T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-31T00:00:00+00:00'
[2025-06-01T00:00:05.718+0000] {python.py:201} INFO - Done. Returned value was: /opt/***/data/raw/synthetic_batch_1.parquet
[2025-06-01T00:00:05.771+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=real_estate_pipeline, task_id=generate_data, execution_date=20250531T000000, start_date=20250601T000004, end_date=20250601T000005
[2025-06-01T00:00:05.828+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-06-01T00:00:05.924+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
